ci/build_python.sh

```diff
@@ -13,15 +13,10 @@ export CMAKE_GENERATOR=Ninja
 
 rapids-print-env
 
-package_name="cuml"
-package_dir="python"
-
 version=$(rapids-generate-version)
-git_commit=$(git rev-parse HEAD)
 export RAPIDS_PACKAGE_VERSION=${version}
```

Comment:
We changed up how this works for some other packages, to something like `rapids-generate-version > ./VERSION`. Does this need a similar change?

e.g. https://github.com/rapidsai/cudf/blob/branch-24.08/ci/build_python.sh

Reply:
Yes I think so, you're right. I'll change that here.

Reply:
I just pushed https://github.com/rapidsai/cuml/pull/5804/commits/29911b623158edacbff8b5ce61424cb711bc3d2a doing that.

Made a similar change in `build_cpp.sh` as well (which doesn't need the version to be written to the `VERSION` file).